opentelemetry-sdk/src/metrics/internal/aggregator.rs

```diff
@@ -15,6 +24,11 @@ pub(crate) trait Aggregator<T>: Send + Sync {
     fn as_precompute_aggregator(&self) -> Option<Arc<dyn PrecomputeAggregator<T>>> {
         None
     }
+
+    /// Checks whether aggregator has hit cardinality limit for metric streams
+    fn check_stream_cardinality(&self, size: usize) -> bool {
```

Comment:
nit: functions with `check` tend to suggest a `Result` return type, maybe have the name be something along the lines of `is_valid_stream_cardinality` or `is_under_cardinality_limit`, etc?